### PR TITLE
json dumps the analysis results

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR uses json.dumps to be sure result is in sensible download format.

### CODE REVIEW

File: skyportal/handlers/api/analysis.py [hunk 1]
```diff
@@ -1392,6 +1392,7 @@ async def get(
                             await self.send_file(
                                 output_data, filename, output_type=output_type
                             )
+                            return
```
Comment (guynir42) [inline skyportal/handlers/api/analysis.py:1395]: This return statement seems out of place...

Comment (mcoughlin) [reply skyportal/handlers/api/analysis.py:1395]: It's not. It sends the file and returns.

Comment (guynir42) [reply skyportal/handlers/api/analysis.py:1395]: Shouldn't this be a `retutn.success`?

Comment (mcoughlin) [reply skyportal/handlers/api/analysis.py:1395]: That returns json, which is not what we want.

Comment (guynir42) [reply skyportal/handlers/api/analysis.py:1395]: I don't think we are supposed to have a handler that doesn't return either success or failure.

Comment (mcoughlin) [reply skyportal/handlers/api/analysis.py:1395]: I think the file download stuff is an exception here.

Comment (guynir42) [reply skyportal/handlers/api/analysis.py:1395]: So if `send_file` replaces this action of passing back `success`, what happens in other cases where there isn't this `return` statement right after it?

Comment (guynir42) [reply skyportal/handlers/api/analysis.py:1395]: ok I see how it works. This is fine

**Review by guynir42 — APPROVED** [review]

**Pull request merged**